### PR TITLE
[docs] Clarify confusion

### PR DESCRIPTION
**Pull request by oliviertassinari**
The license is for MUI X Pro, not MUI X.

### CODE REVIEW

**Comment by oliviertassinari** [comment]
Off-topic, the review might feel weird:

Screenshot 2021-09-02 at 21 33 58

The pakcage is @mui/x-license-pro but the folder is x-license. I propose we rename it to mui*-x-license-pro*.

**Comment by oliviertassinari** [comment]
@danilo-leal Regarding the URLs, what should I use? Are we going with http://mui.com/x/ or http://mui.com/x-advanced/? And what should be the URL of the license page #1865?

**Review by mbrookes — APPROVED** [review]

**Comment by danilo-leal** [comment]
> @danilo-leal Regarding the URLs, what should I use? Are we going with http://mui.com/x/ or http://mui.com/x-advanced/? And what should be the URL of the license page #1865?

I believe that the best way to go, after [revisiting this discussion on the man website pages refinement PR](https://github.com/mui-org/material-ui/pull/28081#discussion_r701443239), is `mui.com/x`. We can always use the _"advanced"_ word as a main characteristic/attribute of the components, but as far as the name of the product goes, it seems we're better off naming it as MUI X. The benefit comes whenever we have to mention the paid version of the components (MUI X _[Pro or Premium]_), which is way clearer and less redundant than MUI X Advanced Pro (or Premium).

No issues with `/mui-x-license-pro/`, as proposed, for the license, though. Looks good to me!

**Comment by oliviertassinari** [comment]
> No issues with /mui-x-license-pro/, as proposed, for the license, though. Looks good to me!

Haha, yeah, it's for the name of the folder on GitHub 😁

> mui.com/x

Ok, no objections, I was wondering about either it would be too short for an URL or not. I assume that we will only try to get this page rank for the MUI X query, so we should be good.

File: packages/x-license/LICENSE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# [Material-UI X](https://material-ui.com/x/)
+# [Material-UI X Pro](https://material-ui.com/x-advanced/)
```
Comment (oliviertassinari) [inline packages/x-license/LICENSE.md:1]:
```suggestion
# [Material-UI X Pro](https://material-ui.com/x/)
```

File: packages/x-license/LICENSE.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
-# [Material-UI X](https://material-ui.com/x/)
+# [Material-UI X Pro](https://material-ui.com/x/)
```
Comment (oliviertassinari) [inline packages/x-license/LICENSE.md:1]: Actually, this might be even clearer

```suggestion
# [Material-UI X Pro](https://material-ui.com/pricing/)
```

Comment (danilo-leal) [reply packages/x-license/LICENSE.md]: Yeah, maybe! `/x` will still exist though, for being the marketing page for the advanced components. But if we were meaning to have one page especially for them in the paid version, I guess linking right to the pricing one might do the trick yes!

**Pull request merged**